src/elm/Page/Transfer.elm

```diff
@@ -0,0 +1,414 @@
+
+        model =
+            { status = currentStatus }
+    in
+    ( model, initCmd shared currentStatus )
+
+
+initCmd : Shared -> Status -> Cmd Msg
+initCmd shared status =
+    case status of
+        Loading transferId ->
+            Api.Graphql.query shared (transferQuery transferId) CompletedTransferLoad
+
+        _ ->
+            Cmd.none
+
+
+
+-- SUBSCRIPTIONS
+
+
+subscriptions : Model -> Sub Msg
+subscriptions _ =
+    Sub.none
+
+
+
+-- MODEL
+
+
+type alias Model =
+    { status : Status
+    }
+
+
+type State
+    = Transferred
+    | Received
+    | NotInvolved -- if the user looking at the transaction is not involved
+
+
+type Status
+    = Loading Int
+    | InvalidId String
+    | LoadFailed (Graphql.Http.Error (Maybe Transfer))
+    | Loaded (Maybe Transfer) State
+
+
+
+-- VIEW
+
+
+view : LoggedIn.Model -> Model -> Html Msg
+view loggedIn model =
+    let
+        t =
+            I18Next.t loggedIn.shared.translations
+    in
+    case model.status of
+        Loading _ ->
+            Page.fullPageLoading
+
+        InvalidId invalidId ->
+            div [ class "container mx-auto px-4" ]
+                [ Lazy.lazy viewHeader loggedIn
+                , div []
+                    [ text (invalidId ++ t "transfer_result.errors.invalid_id") ]
+                ]
+
+        LoadFailed error ->
+            div []
+                [ viewHeader loggedIn
+                , Page.fullPageGraphQLError (t "transfer_result.title") error
+                ]
+
+        Loaded maybeTransfer state ->
+            case maybeTransfer of
+                Just transfer ->
+                    div []
+                        [ Lazy.lazy viewHeader loggedIn
+                        , div [ class "container" ]
+                            [ viewDoggo loggedIn transfer state
+                            , viewCommunity loggedIn transfer
+                            ]
+                        ]
+
+                Nothing ->
+                    div [ class "container mx-auto px-4" ]
+                        [ div []
+                            [ text "Could not load the transfer" ]
+                        ]
+
+
+viewHeader : LoggedIn.Model -> Html Msg
+viewHeader ({ shared } as loggedIn) =
+    let
+        t =
+            I18Next.t loggedIn.shared.translations
+    in
+    div [ class "h-16 w-full bg-indigo-500 flex px-4 items-center" ]
+        [ a
+            [ class "items-center flex absolute"
+            , Route.href Route.Dashboard
+            ]
+            [ Icons.back ""
+            , p [ class "text-white text-sm ml-2" ]
+                [ text (I18Next.t shared.translations "back")
+                ]
+            ]
+        , p [ class "text-white mx-auto" ] [ text (t "transfer_result.title") ]
+        ]
+
+
+viewDoggo : LoggedIn.Model -> Transfer -> State -> Html Msg
+viewDoggo loggedIn transfer state =
+    let
+        t =
+            I18Next.t loggedIn.shared.translations
+    in
+    div [ class "static flex" ]
```

Comment:
Please remove the `static` class. According to Tailwind docs:

> Static
> 
> Use .static to position an element according to the normal flow of the document.
> Any offsets will be ignored and the element will not act as a position reference for absolutely positioned children.

So it only makes sense for elements with absolute children and elements that have a specific x/y position on the screen. Since we have no absolute child and this section grows with `w-full` and the h is dependent on the content so the static element of it will not be different from not being static